compiler/passes/scopeResolve.cpp

```diff
@@ -1790,7 +1790,7 @@ static bool lookupThisScopeAndUses(const char*           name,
         forv_Vec(UseStmt, use, *moduleUses) {
           if (use != NULL) {
             ModuleSymbol* modSymMatches = NULL;
```

Comment:
Shouldn't modSymMatches now also be removed since it's always NULL?

Reply:
Thank you for catching that!